package.json

```diff
@@ -12,11 +12,14 @@
   },
   "devDependencies": {
     "@types/node": "^18.14.1",
+    "prettier": "^3.3.3",
     "typescript": "^4.9.5"
   },
   "scripts": {
     "build": "tsc",
-    "start": "bun ./dist/index.js"
+    "start": "bun ./dist/index.js",
+    "check": "npx prettier . --check",
```

Comment:
@igprad minor fix: use `bunx` instead of `npx` so it can be run inside build (bun) container 🫡

Reply:
Actually, I tried to used `bun prettier . --write` on my local machine, but somehow doesn't work. May you try this in your local container just to make sure that using bun also works?

Reply:
I see, i missed that you mention `bunx` instead of `bun`.
Tried on my local, it working, will adjust this.

Reply:
updated